baybe/surrogates/gaussian_process/presets/default.py

```diff
@@ -0,0 +1,110 @@
+"""Default preset for Gaussian process surrogates."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from attrs import define
+
+from baybe.kernels.basic import MaternKernel, ScaleKernel
+from baybe.priors.basic import GammaPrior
+from baybe.surrogates.gaussian_process.kernel_factory import KernelFactory
+
+if TYPE_CHECKING:
+    from torch import Tensor
+
+    from baybe.kernels.base import Kernel
+    from baybe.searchspace.core import SearchSpace
+
+
+@define
+class DefaultKernelFactory(KernelFactory):
+    """A factory providing the default kernel for the Gaussian process surrogate.
+
+    The logic is adapted from EDBO (Experimental Design via Bayesian Optimization).
+
+    References:
+    *   https://github.com/b-shields/edbo
+    *   https://doi.org/10.1038/s41586-021-03213-y
+    """
+
+    def __call__(  # noqa: D102
+        self, searchspace: SearchSpace, train_x: Tensor, train_y: Tensor
+    ) -> Kernel:
+        # See base class.
+
+        mordred = (searchspace.contains_mordred or searchspace.contains_rdkit) and (
+            train_x.shape[-1] >= 50
+        )
+
+        # low D priors
+        if train_x.shape[-1] < 10:  # <-- different condition compared to EDBO
+            lengthscale_prior = GammaPrior(1.2, 1.1)
+            lengthscale_initial_value = 0.2
+            outputscale_prior = GammaPrior(5.0, 0.5)
+            outputscale_initial_value = 8.0
+
+        # DFT optimized priors
+        elif mordred and train_x.shape[-1] < 100:
+            lengthscale_prior = GammaPrior(2.0, 0.2)
+            lengthscale_initial_value = 5.0
+            outputscale_prior = GammaPrior(5.0, 0.5)
+            outputscale_initial_value = 8.0
+
+        # Mordred optimized priors
+        elif mordred:
+            lengthscale_prior = GammaPrior(2.0, 0.1)
+            lengthscale_initial_value = 10.0
+            outputscale_prior = GammaPrior(2.0, 0.1)
+            outputscale_initial_value = 10.0
+
+        # OHE optimized priors
+        else:
+            lengthscale_prior = GammaPrior(3.0, 1.0)
+            lengthscale_initial_value = 2.0
+            outputscale_prior = GammaPrior(5.0, 0.2)
+            outputscale_initial_value = 20.0
+
+        return ScaleKernel(
+            MaternKernel(
+                nu=2.5,
+                lengthscale_prior=lengthscale_prior,
+                lengthscale_initial_value=lengthscale_initial_value,
+            ),
+            outputscale_prior=outputscale_prior,
+            outputscale_initial_value=outputscale_initial_value,
+        )
+
+
+def _default_noise_factory(
```

Comment:
is the noise always strictly separated from the kernel or why does it get its own factory?

Reply:
I guess this thread is now obsolete since we already clarified it [here](https://github.com/emdgroup/baybe/pull/235#discussion_r1601190539)? Short summary: utlimately, we want to enable user to configure likelihoods and kernels (and perhaps even mean functions) separately, in a composable way. Will add the likelihood mechanism on top of the existing commits. Feel free to resolve unless further discussion needed